enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -96,6 +94,7 @@ func (i *InvocationStatService) getTrendBasicQuery(timezoneOffsetMinutes int32)
 
 	q = q + `
 	    COUNT(1) AS total_num_builds,
+			SUM(CASE WHEN duration_usec > 0 THEN duration_usec END) as total_build_time_usec,
```

Comment:
nit: the tab/spaces are inconsistent.

Reply:
hahahah sorry, i think i have this fixed in all of my editor environments now.